### PR TITLE
added features dccr_ftr_config, dccr_ftr_workshop

### PR DESCRIPTION
**Pull request by rigoucr**

### CODE REVIEW

**Comment by kporras07** [comment]
Published by default
Flag texts
Flag permissions

File: sites/all/modules/custom/dccr/dccr.install [hunk 1]
```diff
@@ -58,6 +58,7 @@ function dccr_install() {
     'i18n_user',
     'i18n_variable',
     'i18n_block',
+
```
Comment (kporras07) [inline sites/all/modules/custom/dccr/dccr.install]: This line shouldn't exist :)

File: sites/all/modules/custom/dccr/dccr.install [hunk 2]
```diff
@@ -70,3 +71,22 @@ function dccr_update_7000() {
...
+    'ds_extras',
+    'flag',
+    'translation_helpers',
+    'admin_menu_toolbar',
+    'dccr_ftr_config',
+    'dccr_ftr_workshop',
+  );
+  module_enable($module_list, TRUE);
+}
+
```
Comment (kporras07) [inline sites/all/modules/custom/dccr/dccr.install]: This line shouldn't exist :)

File: sites/all/modules/custom/dccr/dccr.install [hunk 3]
```diff
@@ -70,3 +71,22 @@ function dccr_update_7000() {
   $module_list = array('dccr_ftr_users');
   module_enable($module_list, TRUE);
 }
+
+/**
+ * Enable ds, ds_extras, date_views, date_popup, flag, admin_menu_toolbar, translation_helpers, modules and features dccr_ftr_workshop, dccr_ftr_config
+ */
+function dccr_update_7001(){
```
Comment (kporras07) [inline sites/all/modules/custom/dccr/dccr.install]: Space after parenthesis

File: sites/all/modules/custom/dccr/dccr.install [hunk 4]
```diff
@@ -70,3 +71,22 @@ function dccr_update_7000() {
   $module_list = array('dccr_ftr_users');
   module_enable($module_list, TRUE);
 }
+
+/**
+ * Enable ds, ds_extras, date_views, date_popup, flag, admin_menu_toolbar, translation_helpers, modules and features dccr_ftr_workshop, dccr_ftr_config
```
Comment (kporras07) [inline sites/all/modules/custom/dccr/dccr.install]: Comment should finish with full stop (.)

File: sites/all/modules/custom/dccr/dccr.install (hunk 2 above)
Comment (rigoucr) [inline sites/all/modules/custom/dccr/dccr.install]: I do not know which line should not go

Comment (kporras07) [inline sites/all/modules/custom/dccr/dccr.install]: empty line

Comment (rigoucr) [inline sites/all/modules/custom/dccr/dccr.install]: ok ok thanks

**Pull request merged**